.gitignore

```diff
@@ -1 +1,4 @@
-.idea
\ No newline at end of file
+.idea
+mol.csv.gz
```

Comment:
Może lepiej katalog `data` i to tam wrzucić?

Reply:
ok